### PR TITLE
Back to top button for quick navigate to top (#6595)

### LINKED ISSUE #6595: Back to top button for quick navigate to top
**Is your feature request related to a problem? Please describe.**
we can add Back to top button for quick navigate to top and this button help to quick navigate to top especially in desktop. You can add here the Back to top button see in the screenshot for the better understanding.

**Additional context**

### PR DESCRIPTION
**Pull request by mehakkkk**
Fixes #6595 

#### Short description of what this resolves:
Button added in the footer component to scroll back to the top of the page.

#### Changes proposed in this pull request:

-Button added in footer-main.hbs with action scrollTop
-Function scrollTop implemented in footer-main.js

ezgif com-gif-maker

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks. commented the below in the issue. https://github.com/fossasia/open-event-frontend/issues/6595#issuecomment-787490549

Expected would be and arrow up button that scrolls the page to the top. Please compare for example to https://2019.fossasia.org/

Screenshot from 2021-02-28 18-44-30

**Comment by mehakkkk** [comment]
@mariobehling changes are updated, please review this PR sir

**Comment by MeghalBisht** [comment]
A question mark appears
Screenshot from 2021-03-01 14-07-49

File: app/templates/components/footer-main.hbs [hunk 1]
```diff
@@ -130,6 +131,10 @@
             {{/each-in}}
           </div>
         </UiDropdown>
+        
+      </div>
+      <div class="three wide computer four wide tablet column right aligned" style="float: right;">
+        <p class="mybtn" title ="Back to top" {{action "scrollTop"}}>&#11165; </p>
```
Comment (MeghalBisht) [inline app/templates/components/footer-main.hbs:137]: semantic ui icon should be used inside a button tag

Comment (maze-runnar) [inline app/templates/components/footer-main.hbs:137]: `<i class="some icon"></i>` use semantic ui icon. https://semantic-ui.com/elements/icon.html

File: app/styles/partials/footer.scss [hunk 2]
```diff
@@ -32,6 +32,23 @@ footer {
...
+  z-index: 99;
+  font-size: 25px;
+  border: none;
+  background-color: transparent;
+  outline: none;
+  color: rgb(255, 255, 255);
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 70;
+}
```
Comment (maze-runnar) [inline app/styles/partials/footer.scss:50]: does this much css really needed.

**Comment by mehakkkk** [comment]
@maze-runnar @MeghalBisht Thanks for reviewing, will make the required changes

**Review by Naman-1234 — COMMENTED** [review]
Try to make the button icon a little bit smaller and also change the color of it as it is not visible in the white screen portion.

File: app/templates/components/footer-main.hbs [hunk 3]
```diff
@@ -130,6 +131,10 @@
             {{/each-in}}
           </div>
         </UiDropdown>
+        
+      </div>
+      <div class="floating labeled grey search icon mini button">
+        <button class="scroll-button"  title ="Back to top" {{action "scrollTop"}}><i class="chevron up icon"></i></button>
```
Comment (MeghalBisht) [inline app/templates/components/footer-main.hbs:137]: Please use semantic ui icons. Instead of creating one.

**Comment by MeghalBisht** [comment]
No response

**Pull request closed without merging**